src/templates/blog-post.tsx

```diff
@@ -58,25 +68,72 @@ const BlogHeader = ({ frontmatter }) => {
   );
 };
 
+const HeroImageDefault = styled(GatsbyImage)`
+  display: block;
+  @media (max-aspect-ratio: 0.75) {
+    display: none;
+  }
+`;
+
+const HeroImageSquared = styled(GatsbyImage)`
+  display: none;
+  @media (max-aspect-ratio: 0.75) {
+    display: block;
+  }
+`;
+
+const BlogHeroImage = ({ wideImage, squareImage }) => {
+  /**
+   * A screen is tall once the height is larger then the width, which means the aspect ratio dips below 1 (0-1)
```

Comment:
The comment says "A screen is [considered as] tall..." but I don't see any mentioning of tallness in the code below. As a casual reader this puzzles me a bit, maybe the comment could rather explain that the output of either one of HeroImageDefault or HeroImageSquared is visually hidden depending on the viewport aspect ratio. This gets apparent by looking at the styled components directly above, but giving a reference to "aspect-ratio" rather than "tallness" might save the reader a second or two.